.github/workflows/authors.yml

```diff
@@ -1,5 +1,9 @@
 name: "authors update"
 on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: '5 4 * * *'
```

Comment:
Isn't this "every day at 04:05" ?
https://crontab.guru/#5_4_*_*_*

Reply:
I updated the comment but didn't update the cron entry I copy/pasted from a different workflow. Whoops. Updating....